<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRandomParcellationViewControls</class>
 <widget class="QWidget" name="QmitkRandomParcellationViewControls">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>249</width>
    <height>517</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QmitkTemplate</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelWarning">
     <property name="styleSheet">
      <string notr="true">QLabel { color: rgb(255, 0, 0) }</string>
     </property>
     <property name="text">
      <string>Please select an image!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxNumberNodes">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Nodes</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Choose the number of nodes:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxNumberNodes">
        <property name="minimum">
         <number>2</number>
        </property>
        <property name="maximum">
         <number>5000</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxMerging">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Merging</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="checkBoxMerging">
        <property name="text">
         <string>Merging on</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frameMerging">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QRadioButton" name="radioButtonNumberParcels">
           <property name="text">
            <string>According to the number of parcels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameNumberParcels">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Number of Parcels:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxNumberParcels">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>5000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSmallestParcel">
           <property name="text">
            <string>According to the size of parcels</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frameSmallestParcel">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Size of the smallest parcel:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxSmallestParcel">
              <property name="minimum">
               <number>2</number>
              </property>
              <property name="maximum">
               <number>100000</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxJustMergeSmallParcels">
              <property name="text">
               <string>Just merge small parcels</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buttonSelectRandomNodes">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Do image processing</string>
     </property>
     <property name="text">
      <string>Select random nodes</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxCounting">
     <property name="title">
      <string>Counting</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="labelNumberVoxels">
        <property name="text">
         <string>Number of voxels with value 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelNumberNodes">
        <property name="text">
         <string>Number of nodes:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
